### ISSUE TITLE
Does it support social authentication?

### ISSUE THREAD

**Issue by akshaykatale99** [wp-graphql/wp-graphql-jwt-authentication#125]
Does this plugin have any support for social login like Facebook and google? is it feasible to extend this plugin to have support for social auth.

Thanks

**Comment by therealgilles** [comment]
I don't think so. This plugin is meant at doing authentication with the WP backend and use the WP user database.

Most "social" logins requires OAuth2, which is different from JWT authentication. It would be best to use external services like Auth0, Firebase, or Passport. Not sure if it's then possible to easily use that for user authentication on the WP backend. Maybe it will require some integration with the service respective WP plugin.

**Comment by scottyzen** [comment]
Would love to see a solution for this. I've tried a couple of things but have not quite figured it out yet.

**Comment by justlevine** [comment]
If anyone here is still looking for a solution, I just started [beta testing](https://wp-graphql.slack.com/archives/C3NM1M291/p1666091193903229) an OAuth2 Login extension for WPGraphQL, and could use some people to put it through the paces before the repo is release publicly in a couple months. If you're interested lemme know in Slack, and I'll circle back to share the GH link once its released.

**Comment by ryntab** [comment]
You can find the WPGraphQL OAuth2 extension here: https://github.com/AxeWP/wp-graphql-headless-login

**Comment by stephane-segning** [comment]
> You can find the WPGraphQL OAuth2 extension here: https://github.com/AxeWP/wp-graphql-headless-login

I think this issue can now be close
